src/content_scripts/autocorrect.js

```diff
@@ -0,0 +1,351 @@
+"use strict";
+
+const fractions = Object.freeze(["¼", "½", "¾", "⅐", "⅑", "⅒", "⅓", "⅔", "⅕", "⅖", "⅗", "⅘", "⅙", "⅚", "⅛", "⅜", "⅝", "⅞"]);
+
+const fractionValues = Object.freeze([1.0 / 4.0, 1.0 / 2.0, 3.0 / 4.0, 1.0 / 7.0, 1.0 / 9.0, 1.0 / 10.0, 1.0 / 3.0, 2.0 / 3.0, 1.0 / 5.0, 2.0 / 5.0, 3.0 / 5.0, 4.0 / 5.0, 1.0 / 6.0, 5.0 / 6.0, 1.0 / 8.0, 3.0 / 8.0, 5.0 / 8.0, 7.0 / 8.0]);
```

Comment:
I made it an array since it is easier to loop though below, but I will change it to a map.

Reply:
Hmm, what do you mean by easier? If you need to iterate it and performance matters, I'd suggest to [use a real Map object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Map).

Reply:
Oh, this is the same issue as with the communication types below, since we cannot load modules in content scripts. Also, we do not need the rest of the Symbols file, so no need to waste the RAM.